client/src/components/viewers/FileViewerActionBar.vue

```diff
@@ -0,0 +1,43 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ms-action-bar
+    class="file-viewer-action-bar"
+  >
+    <ms-action-bar-button
+      v-for="action, key in actions"
+      :key="key"
+      :icon="action.icon"
+      @click="action.handler"
+      :button-label="action.text"
+    />
+  </ms-action-bar>
+</template>
+
+<script setup lang="ts">
+import { MsActionBar, MsActionBarButton, Translatable } from 'megashark-lib';
+import { PropType } from 'vue';
+
+interface FileViewerAction {
```

Comment:
You'll probably want a `disabled` attribute to prevent clicking in some cases (max zoom level reached for example).
Also better to extract this interface to a separate file to be able to import it for typing.

Reply:
Since the action bar will change depending on the feature list we will make with Fabien on monday, I just add your suggestions as comment to keep it in mind in the next PR.